pkg/extensions/enterprise_imports.go

```diff
@@ -6,6 +6,7 @@ package extensions
 import (
 	_ "cloud.google.com/go/kms/apiv1"
 	_ "cloud.google.com/go/kms/apiv1/kmspb"
+	_ "cloud.google.com/go/spanner"
```

Comment:
I think your Enterprise version might be old, because these imports shouldn't be here